File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {

     

        for(int i = 0 ; i < nums.length; i++){
           int temp = nums[i];
         
            for(int j = i + 1  ; j < nums.length ; j++){
                System.out.println(temp);
                System.out.println(nums[j]);
                if (temp == nums[j]){
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:

        import heapq

        table = dict()
        for num in nums:
            if num in table:
                table[num] += 1
            else:
                table[num] = 1

        heap = []
        for key, value in table.items():
            ele = (value, key)
            if len(heap) < k:
                heapq.heappush(heap, ele)
            else:
                heapq.heappushpop(heap, ele)

        answer = []
        for keys, value in heap:
            answer.append(value)
            
        return answer


        